oe_theme.theme

```diff
@@ -807,3 +807,24 @@ function oe_theme_preprocess_responsive_image_formatter(&$variables) {
   $variables['responsive_image']['#attributes']['class'][] = 'ecl-image';
   $variables['responsive_image']['#attributes']['class'][] = 'ecl-image--fluid';
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_contextual_navigation(array &$variables): void {
```

Comment:
Could you move this below the rest of the preprocessors that deal with the paragraph types and adjust the docblock to be consistent with the rest? Thanks!